src/mantle/util.c

```diff
@@ -38,13 +38,13 @@ VkFormat getVkFormat(
     case PACK_FORMAT(GR_CH_FMT_R4G4B4A4, GR_NUM_FMT_UNORM):
         break; // FIXME VK_FORMAT_A4B4G4R4_UNORM_PACK16 isn't available
     case PACK_FORMAT(GR_CH_FMT_R5G6B5, GR_NUM_FMT_UNORM):
-        return VK_FORMAT_B5G6R5_UNORM_PACK16;
-    case PACK_FORMAT(GR_CH_FMT_B5G6R5, GR_NUM_FMT_UNORM):
         return VK_FORMAT_R5G6B5_UNORM_PACK16;
+    case PACK_FORMAT(GR_CH_FMT_B5G6R5, GR_NUM_FMT_UNORM):
+        return VK_FORMAT_B5G6R5_UNORM_PACK16;
     case PACK_FORMAT(GR_CH_FMT_B5G6R5, GR_NUM_FMT_SRGB):
         break; // FIXME VK_FORMAT_R5G6B5_SRGB_PACK16 isn't available
     case PACK_FORMAT(GR_CH_FMT_R5G5B5A1, GR_NUM_FMT_UNORM):
-        break; // FIXME VK_FORMAT_A1B5G5R5_UNORM_PACK16 isn't available
+	return VK_FORMAT_R5G5B5A1_UNORM_PACK16;
```

Comment:
Yes it is, didn't notice that.

Reply:
ok, that's fixed now

Reply:
Not the maintainer, of course, but usually in a context like this you'd merge such fixes into relevant previous commit by using git rebase and then force pushing that instead of having a separate fixup commit.

Reply:
yeah the tab changes are polluting the diff in this case. would be nice if you could fix that (use `git rebase -i`)

Reply:
ok, done